.github/workflows/ci.yml

```diff
@@ -1,59 +1,29 @@
-name: CI
+name: "CI"
```

Comment:
Their CI is pretty outdated. Ideally they would fix it upstream but they've been ignoring PRs. Also, I originally wanted to add jobs for mac and windows but couldn't get either of them to work in CI. I also wanted to add a prettier job but this repo is not compatible with prettier.

I guess I can revert the changes since my experiments failed. We do want to add a Dependabot auto-merge job but I'll do that in a separate file.